csi.proto

```diff
@@ -1318,6 +1393,10 @@ message NodeServiceCapability {
       GET_VOLUME_STATS = 2;
       // See VolumeExpansion for details.
       EXPAND_VOLUME = 3;
+      // If Plugin implements VOLUME_HEALTH capability
+      // then it MUST implement NodeGetVolumeStats RPC
+      // call for fetching volume health information.
+      VOLUME_HEALTH = 4 [(alpha_enum_value) = true];
```

Comment:
Ditto. Should we change this to `VOLUME_CONDITION`?

Reply:
Done